### PR TITLE
Small fixes to early game lines

### PR DESCRIPTION
**Pull request by ttgmichael**
Without nl's
withoutnl3
withoutnl2
withoutnl
With nl's
withnl
withnl2
withnl3
withnl5

Problem with nl D:
withnl4

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -2400,7 +2400,7 @@ QUEST_LV_0100_20150317_002395	$I'll wait then
 QUEST_LV_0100_20150317_002396	$About the monsters
 QUEST_LV_0100_20150317_002397	Talk to the Battle Commander (2)
 QUEST_LV_0100_20150317_002398	$Accept
-QUEST_LV_0100_20150317_002399	$Start of the Trouble
+QUEST_LV_0100_20150317_002399	$The Beginning of Trouble
```
Comment (vtange) [inline QUEST_LV_0100.tsv:2403]: I have $The Start of Trouble

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -8273,11 +8273,11 @@ QUEST_LV_0100_20151001_008268	That fellow knows well about the purifiers in the
 QUEST_LV_0100_20151001_008269	{memo X}You want to learn my skills?{nl}It's not hard for me to teach you, but would you be able to handle it?
 QUEST_LV_0100_20151001_008270	The Goddess always told us to help those around us.{nl}By helping others near you, you will help your spirit as well.
 QUEST_LV_0100_20151001_008271	Great King Zachariel's ordeal will not be easy.{nl}With the skill you have, he should have acknowledged you.
-QUEST_LV_0100_20151001_008272	The kingdom has lost all its hope. {nl}It does not have any will to stand back anymore and is just crying out for the Goddess.{nl}
-QUEST_LV_0100_20151001_008273	It is during this time that the people came forward saying they dreamed of the Goddess. {nl}It is you, the Revelator. {nl}
+QUEST_LV_0100_20151001_008272	Our kingdom has lost all its hope. {nl}It does not have any will to stand back anymore and is just crying out for the Goddesses.{nl}
+QUEST_LV_0100_20151001_008273	And from such chaotic times, there are now people who come forward stating that they've dreamed of the Goddess. {nl}People such as yourself are the Revelators. {nl}
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: i have 8276 and 8277 covered in #1279

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -8119,12 +8119,12 @@ QUEST_LV_0100_20150918_008114	The Writings on the Wall
...
-QUEST_LV_0100_20150918_008118	Thank you for passing Laimonas' regards.{nl}You should enter Klaipeda.{memo Uska is a knight, keep the Sir} Sir Uska is waiting for the Revelators.
+QUEST_LV_0100_20150918_008118	Thank you for completing Laimonas' favor.{nl}You may enter Klaipeda.{memo Uska is a knight, keep the Sir} Sir Uska is waiting for the Revelators.
 QUEST_LV_0100_20150918_008119	$There might be more of these Large Kepas roaming in the area nearby.{nl}Could you take a look for me, please?{nl}
 QUEST_LV_0100_20150918_008120	$It was just as I thought, right? It was this instinct that kept me alive during Medzio Diena.
 QUEST_LV_0100_20150918_008121	$It's been too long, my men should've come back, but I haven't heard anything from them yet.{nl}I know it's been four years since Medzio Diena, but my men have been getting too relaxed recently.
 QUEST_LV_0100_20150918_008122	
-QUEST_LV_0100_20150918_008123	The Goddesses disappearing, things like golems appearing... {nl}Is there really any hope?
+QUEST_LV_0100_20150918_008123	With the Goddesses disappearing, and monsters like the golems appearing... {nl}Is there really any hope?
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: golems -> Golem

(It reads better when you say: "and monsters like the Golem appearing.." 

also, you encounter one.

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Alrighty, I'll just take care of the merge conflict when it comes. xD Your's sound better

**Comment by ttgmichael** [comment]
@Sunaries see original post! :)

Also will revert changes to knight

Have also reverted the nl changes (since the buttons look off)

if only we could have the best of both worlds :S

**Comment by Sunaries** [comment]
It's a positive change except for that specific problem that you've already addressed.

Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**